pkg/providers/snow/objects.go

```diff
@@ -32,10 +31,10 @@ func ControlPlaneObjects(ctx context.Context, clusterSpec *cluster.Spec, kubeCli
 	}
 	capiCluster := CAPICluster(clusterSpec, snowCluster, kubeadmControlPlane)
 
-	return []runtime.Object{capiCluster, snowCluster, kubeadmControlPlane, new}, nil
+	return []kubernetes.Object{capiCluster, snowCluster, kubeadmControlPlane, new}, nil
 }
 
-func WorkersObjects(ctx context.Context, clusterSpec *cluster.Spec, kubeClient kubernetes.Client) ([]runtime.Object, error) {
+func WorkersObjects(ctx context.Context, clusterSpec *cluster.Spec, kubeClient kubernetes.Client) ([]kubernetes.Object, error) {
```

Comment:
nit: should this be named `WorkerObjects` like the `ControlPlaneObjects` above?

Reply:
ummm I don't have a strong opinion
I think the idea was that there is only one CP but possibly multiple workers